client/java/generator/src/main/java/io/openlineage/client/JavaPoetGenerator.java

```diff
@@ -86,9 +87,31 @@ public void generate(PrintWriter printWriter) throws IOException {
         .addModifiers(PUBLIC, FINAL);
 
     if (!server) {
+      containerTypeBuilder.addJavadoc(
+          "Usage:\n" +
+          "<pre>\n" +
+          "  URI producer = URI.create(\"http://my.producer/uri\");\n" +
+          "  $N ol = new $N(producer);\n" +
+          "  UUID runId = UUID.randomUUID();\n" +
+          "  RunFacets runFacets =\n" +
+          "    ol.newRunFacetsBuilder().nominalTime(ol.newNominalTimeRunFacet(now, now)).build();\n" +
+          "  Run run = ol.newRun(runId, runFacets);\n" +
+          "  String name = \"jobName\";\n" +
+          "  String namespace = \"namespace\";\n" +
+          "  JobFacets jobFacets = ol.newJobFacetsBuilder().build();\n" +
+          "  Job job = ol.newJob(namespace, name, jobFacets);\n" +
+          "  List<InputDataset> inputs = Arrays.asList(ol.newInputDataset(\"ins\", \"input\", null, null));\n" +
```

Comment:
`List<InputDataset>`. I think`>` is breaking the code `<pre>` tag which makes the test fail.

Reply:
thanks